var/spack/repos/builtin/packages/cantera/package.py

```diff
@@ -26,132 +26,148 @@
 import os
 
 
-class Cantera(Package):
+class Cantera(SConsPackage):
     """Cantera is a suite of object-oriented software tools for problems
     involving chemical kinetics, thermodynamics, and/or transport processes."""
 
     homepage = "http://www.cantera.org/docs/sphinx/html/index.html"
-    url      = "https://github.com/Cantera/cantera/archive/v2.2.1.tar.gz"
+    url      = "https://github.com/Cantera/cantera/archive/v2.3.0.tar.gz"
 
+    version('2.3.0', 'aebbd8d891cb1623604245398502b72e')
     version('2.2.1', '9d1919bdef39ddec54485fc8a741a3aa')
 
-    variant('lapack',     default=True,
-            description='Build with external BLAS/LAPACK libraries')
-    variant('threadsafe', default=True,
-            description='Build threadsafe, requires Boost')
-    variant('sundials',   default=True,
-            description='Build with external Sundials')
     variant('python',     default=False,
             description='Build the Cantera Python module')
     variant('matlab',     default=False,
             description='Build the Cantera Matlab toolbox')
 
     # Required dependencies
-    depends_on('scons', type='build')
-
-    # Recommended dependencies
-    depends_on('blas',      when='+lapack')
-    depends_on('lapack',    when='+lapack')
-    depends_on('boost',     when='+threadsafe')
-    depends_on('sundials',  when='+sundials')  # must be compiled with -fPIC
```

Comment:
If any of these dependencies were optional for 2.2.1, they are certainly not optional anymore. It seems that each of these dependencies can either be installed externally (`system_fmt=yes`) or you can clone the git repos for them into the Cantera source tree and Cantera will build them for you (`system_fmt=no`). There is no option to build without them.